lambda-notes/src/components/noteForm.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addNewNote, updatedNote } from './actions';
+import { Link } from 'react-router-dom';
+import { Buttons, FormSteez } from './styled-components';
+
+class NoteForm extends React.Component{
+    constructor(props){
+        super(props);
+        this.state ={
+            title: '',
+            textBody: '',
+            updateNote: null,
+            id:''
+        }
+    }
+
+    inputHandler = e =>{
+        this.setState({[e.target.name]: e.target.value})
+    }
+    
+    componentDidMount(){
+        this.setState({
+            title: this.props.title,
+            textBody: this.props.text,
+            updateNote: this.props.update,
+            id: this.props.id
+            
+        })
+    }
+
+     newNote=e=>{
+        const newNote = {
+            title:this.state.title,
+            textBody: this.state.textBody
+        }
+        this.props.addNewNote(newNote);
+    }
+
+    updateNoteHandler = () =>{
+        const newNote = {
+            title:this.state.title,
+            textBody: this.state.textBody
+        }
+        const id = this.state.id
+        this.props.updatedNote(id, newNote)
+    }
+
+
+    render(){
+        return(
```

Comment:
This file could use a lot of descructuring. You are using this.props and this.state over and over again which could be avoided with destructuring.